src/app/(proper_react)/(redesign)/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -46,7 +48,7 @@ const createProfileAndStartScan = async (
 
   const result: WelcomeScanBody = await response.json();
   if (!result?.success) {
-    throw new Error("Could not start scan");
+    logger.error("Could not start scan", { response, status: 403 });
```

Comment:
I don't think logging the status code here is necessary. `response` is probably useful, I think it will be logged correctly since `winston` uses structured logging but we should confirm this in stage (e.g. that it doesn't just end up being `toString()`'d to `Object` or something silly)

Reply:
I would suggest either returning a HTTP `403` to the user here, or redirect them somewhere. If the user isn't eligible we don't want to continue with the `EnterInfo` flow.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4725/commits/5f3f8585817a19c51c47fd6faa11d1775552de6e

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4725/commits/e69a817bbc8f2c66908fb3bfc0aa8ea268686e1a